src/web/extractors.rs

```diff
@@ -758,15 +753,12 @@ impl FromRequest for CollectionPostRequest {
             };
 
             let max_post_records = i64::from(state.limits.max_post_records);
-            let (user_id, db, collection, query, mut bsos) =
-                <(
-                    HawkIdentifier,
-                    Box<dyn Db>,
-                    CollectionParam,
-                    BsoQueryParams,
-                    BsoBodies,
-                )>::from_request(&req, &mut payload)
-                .await?;
+
+            let user_id = HawkIdentifier::from_request(&req, &mut payload).await?;
+            let collection = CollectionParam::from_request(&req, &mut payload).await?;
+            let query = BsoQueryParams::from_request(&req, &mut payload).await?;
+            let mut bsos = BsoBodies::from_request(&req, &mut payload).await?;
```

Comment:
This change (and there's one other instance) can be mostly reverted, since the DB is no longer one of the extractors. This change was made because the DB extractor was not `'static`, while the impl for the tuple of extractors required `'static`.